nowplaying/input/observer.py

```diff
@@ -40,19 +38,27 @@ def __init__(self, current_show_url):
     def add_track_handler(self, track_handler):
         self.track_handler = track_handler
 
-    def handle_id(self, saemubox_id):
-        return False
-
     def update(self, saemubox_id):
         if self.handle_id(saemubox_id):
             self.handle()
 
-    def handle(self):
+    @abstractmethod
+    def handle_id(self, saemubox_id):  # pragma: no coverage
```

Comment:
`handle_is` is being replaced with an abstract method since I could not find an instance where the concrete implementation I'm removing was used.